nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java

```diff
@@ -222,13 +224,29 @@ public void registerContribution(Object contrib, String extensionPoint, Componen
 
     @Override
     public void registerCache(String name, int maxSize, int timeout) {
+        registerCache(name, maxSize, timeout, new CacheDescriptor());
```

Comment:
Why not retrieving the default descriptor ?

Reply:
Actually this method is not used anymore. But since we are specifying the `maxSize` and `timeout`, it could mean that it is not a fallback and that you don't necessarily other characteristics that the default cache could have.

Reply:
What about the `klass` ? could be interesting to have the propre cache wrapper, to handle cluster for instance.